.gitignore

```diff
@@ -1,2 +1,3 @@
 .ci/
-build/
\ No newline at end of file
+build/
+.vscode/launch.json
```

Comment:
You better add this into `~/.gitignore`. That way the file will be ignored in all projects you work on.

Reply:
Thanks for the hint :)